docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/qovery.md

```diff
@@ -5,95 +5,70 @@ description: Learn in this guide how to deploy your Strapi application on Qovery
 
 # Qovery
 
-This is a step-by-step guide for deploying a Strapi project on [Qovery](https://www.qovery.com).
+[Qovery](https://www.qovery.com) is a fully-managed cloud platform that runs on your AWS, Digital Ocean and Scaleway account where you can host static sites, backend APIs, databases, cron jobs, and all your other apps in one place.
 
-Qovery is a CaaS for developers to deploy their full-stack applications on AWS, GCP, Azure and Digital Ocean. Qovery provides access to managed PostgreSQL and free SSL.
+Qovery provides **free hosting** for individual developers and include the following features:
```

Comment:
Would you please remove all this paragraph? We tend to avoid marketing-like material in our documentation :-)